src/core_plugins/getting_started/public/components/getting_started/getting_started.html

```diff
@@ -0,0 +1,180 @@
+            >Elasticsearch API</a>,
+            <a
+              href="{{ gettingStarted.documentationLinks.beats.docs }}"
+              class="kuiLink"
+              target="_blank"
+            >Beats</a>,
+            <a
+              href="{{ gettingStarted.documentationLinks.logstash.docs }}"
+              class="kuiLink"
+              target="_blank"
+            >Logstash</a>.
+            Not sure how to do that?
+            <a
+              href=""
+              class="kuiLink"
+              target="_blank"
+            >Read this introduction. --- LINK NEEDED</a>
+          </p>
+        </div>
+      </div>
+
+      <div class="kuiCard kuiCardGroup__card gettingStartedCard">
+        <div class="kuiCard__description">
+          <div class="kuiCard__descriptionTitle">
+            <h2 class="kuiSubTitle">
+              Visualize and explore
+            </h2>
+          </div>
+
+          <div class="kuiCard__descriptionText gettingStartedCard__descriptionText">
+            <div class="gettingStartedLogoRow">
+              <img
+                class="gettingStartedLogoRow__logo"
+                src="{{ gettingStarted.imageUrls.dashboardIcon }}"
+                alt="Dashboard icon"
+              />
+            </div>
+          </div>
+        </div>
+
+        <div class="kuiCard__footer">
+          <p class="kuiText">Once you've added data, you'll need to
+          <a
+            href="#/management/kibana/index"
+            class="kuiLink"
+            kbn-getting-started-opt-out
+          >configure an index pattern</a>
+          for your data before using our Discover and Visualize applications.</p>
+        </div>
+      </div>
+
+      <div
+        class="kuiCard kuiCardGroup__card gettingStartedCard"
+        ng-show="gettingStarted.hasManageAndMonitorMessages()"
+      >
+        <div class="kuiCard__description">
+          <div class="kuiCard__descriptionTitle">
+            <h2 class="kuiSubTitle">
+              Manage and monitor
+            </h2>
+          </div>
+
+          <div class="kuiCard__descriptionText gettingStartedCard__descriptionText">
+            <div class="gettingStartedLogoRow">
+              <img
+                class="gettingStartedLogoRow__logo"
+                src="{{ gettingStarted.imageUrls.shieldIcon }}"
+                alt="Shield icon"
+              />
+            </div>
+          </div>
+        </div>
+
+        <div
+          class="kuiCard__footer"
+          id="gettingStartedManageAndMonitorMessagesContainer"
+        ></div>
+      </div>
+    </div>
+  </div>
+
+  <div class="kuiViewContentItem kuiVerticalRhythm">
+    <div class="kuiFieldGroup">
+      <div class="kuiFieldGroupSection">
+        <p class="kuiText">Just want to see what Kibana is capable of doing?</p>
+      </div>
+
+      <div class="kuiFieldGroupSection">
+        <a
+          class="kuiButton kuiButton--primary"
+          href="{{ gettingStarted.documentationLinks.demoSite }}"
```

Comment:
Yes, it should be, or else there's a small time window where the expression will not be replaced. Good catch. Thanks!